modular_zubbers/code/modules/cargo/packs/goodies.dm

```diff
@@ -37,6 +37,14 @@
 	access_view = ACCESS_WEAPONS
 	contains = list(/obj/item/gun/energy/e_gun/mini)
 
+/datum/supply_pack/goody/shotgun_revolver
+	name = "Bóbr 12 GA Revolver Single-Pack"
+	desc = "Contains 1 Bóbr revolver, chambered in 12 gauge. For when you really want the power of a shotgun in the palm of your hand. Comes with a box of beanbag shells."
+	contains = list(/obj/item/gun/ballistic/revolver/shotgun_revolver = 1,
+	/obj/item/ammo_box/advanced/s12gauge/buckshot = 1)
+	access_view = ACCESS_WEAPONS
+	cost = PAYCHECK_COMMAND * 15
```

Comment:
```suggestion
	cost = PAYCHECK_COMMAND * 18
```
This is literally cheaper than the actual shotguns you can get, worse or not, it shouldn't be, as it fit in your bag and still hold more shot

Reply:
I was also going to suggest it be more money lol

Reply:
I wasn't certain how much it should be and the person who wanted it added didn't really specify, changed to 18x command paycheck now (1800 credits at this time)